Source/Android/app/src/main/java/org/dolphinemu/dolphinemu/utils/TvUtil.java

```diff
@@ -63,7 +66,16 @@ public static int getNumberOfChannels(Context context)
                             null,
                             null,
                             null);
-    return cursor != null ? cursor.getCount() : 0;
+    if (cursor != null)
+    {
+      channels = cursor.getCount();
+      cursor.close();
```

Comment:
Please use try-with-resources instead.

Reply:
I think I've done this properly now but worth a second look.